cfgov/jinja2/v1/_includes/molecules/related-posts.html

```diff
@@ -8,37 +8,19 @@
 
    Creates related posts markup when given:
 
-   settings:                  An object used to customize the markup.
-
-   settings.posts:            An array of related posts.
-
-   settings.half_width:       A Boolean indicating whether the posts should be
+   half_width:       A Boolean indicating whether the posts should be
                               at half width.
 
-   settings.hide_header_slug: A Boolean indicating whether the header should be
+   hide_header_slug: A Boolean indicating whether the header should be
                               hidden.
 
-   settings.view_more:        An object used to customize the view more markup.
-
    ========================================================================== #}
 
-{% macro render(settings) %}
-{% from 'macros/util/format/url.html' import slugify as slugify %}
-{% set defaults = {
-    'search_types': 'posts, newsroom, press release, speech, testimony',
-    'search_size':  5
-} %}
-{% do settings.update(defaults) %}
-{% set posts = settings.posts or
-               more_like_this(settings.post,
-                              search_types=settings.search_types,
-                              search_size=settings.search_size)
-%}
-
+{% macro render(half_width=false, hide_header_slug=false) %}
 <div class="m-related-posts
             {{ 'm-related-posts__half-width'
                if settings.half_width else '' }}">
```

Comment:
It should be `{% if half_width %}` since it's defined as a local variable and set a default so it will always have a value.